tools/db_bench_tool.cc

```diff
@@ -8185,6 +8207,50 @@ class Benchmark {
     }
   }
 
+  void Backup(ThreadState* thread) {
+    DB* db = SelectDB(thread);
+    std::string backup_dir = FLAGS_db + "/backups";
+    std::unique_ptr<BackupEngineOptions> engine_options_(
```

Comment:
Underscore suffix like in `engine_options_` is usually for private data member (ref: "Data members of classes (but not structs) additionally have trailing underscores." in https://google.github.io/styleguide/cppguide.html#Variable_Names )- we can use `engine_options` here.

Reply:
Updated.